ironic/drivers/modules/drac/raid.py

```diff
@@ -884,6 +972,65 @@ def _check_node_raid_jobs(self, task):
                 self._clear_raid_config_job_failure(node)
                 self._set_clean_failed(task, config_job)
 
+        if finished_job_ids and not node.driver_internal_info.get(
```

Comment:
I think the if statement above should check to see if there are no raid_config_job_ids left instead of checking to see if there was at least 1 job that finished.  Like this:
if not node.driver_internal_info['raid_config_job_ids'] and not node.driver_internal_info.get('raid_config_job_failure'):

Reply:
Done